dali/test/python/test_dali_tf_dataset_shape.py

```diff
@@ -309,11 +301,13 @@ def test_deprecated():
 
 
 def test_deprecated_double_def():
-    yield dali_pipe_deprecated_raises, \
-            { "shapes": 2, "output_shapes": 2, "dtypes": tf.uint8 }, 2, tf.uint8, dali_types.UINT8, 1, 2
-    yield dali_pipe_deprecated_raises, \
+    error_msg = "Usage of `{}` is deprecated in favor of `output_{}`*only `output_{}` should be provided."
+    yield raises(ValueError, error_msg.format(*(("shapes",) * 3)))(dali_pipe_deprecated), \
```

Comment:
maybe 
```
with assert_raises(ValueError, glob=error_msg.format(...)):
   ...
```
would be less of a parentheses rollecoaster?

Reply:
added named variables for error message strings